icons/onion.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="2" />
+  <circle cx="12" cy="12" r="6" />
+  <path d="M2.8 8.1a10 10 0 1 0 5.3-5.3C5 4 3 2 3 2L2 3s2 2 .8 5.1" />
+  <path d="M18 20v2" />
+  <path d="m21 21-1.9-1.9" />
+  <path d="M22 18h-2" />
```

Comment:
I like how it's all nice and geometrical, but... it also doesn't look like the cross section of any onion that grows on Earth.

These rough drafts of mine are admittedly highly inferior in quality and geometrical perfectness, but I still think an onion should look something more like these.
![image](https://github.com/lucide-icons/lucide/assets/17746067/ccd7be5c-4f0a-4ff3-92f7-993f91444d42)